### PR TITLE
Implement Event Reminder Feature

### PR DESCRIPTION
**Pull request by farellano0**
## Description

This pull request introduces a feature that enables users to set a specific date and time for each event to receive an automatic email reminder.

### Changes Made

- Integrated functionality for users to set reminders on events.
- Implemented scheduled email reminders using Action Mailer and Active Job.
- Utilized Sidekiq for background processing of email reminder scheduling and sending.

### Purpose

Enhance user experience by providing a convenient way for users to receive reminders for their events, improving event management and attendance.

## Additional Information

- **Screenshots:**
image

### CODE REVIEW

File: app/views/bright_events/_filters.html.erb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <script>
     function updateMinDate() {
-      var dateFromValue = document.getElementById('date_from').value;
+      let dateFromValue = document.getElementById('date_from').value;
       document.getElementById('date_to').min = dateFromValue;
     }
   </script>
```
Comment (juperameza) [inline app/views/bright_events/_filters.html.erb]: This isn't the way to do J's in rails!!!!

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/jobs/event_reminder_job.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class EventReminderJob < ApplicationJob
+  queue_as :default
+
+  def perform(event_id, event_updated_at)
+    # Do something later
```
Comment (MauricioNC) [inline app/jobs/event_reminder_job.rb]: Remove base code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/bright_events/_filters.html.erb (hunk 1 above)
Comment (JuanequeX) [inline app/views/bright_events/_filters.html.erb]: Move this code to a JavaScript controller using Stimulus. It's not a good practice to include logic in views; separating Ruby or JavaScript logic from views improves maintainability.

**Review by JuanequeX — DISMISSED** [review]

File: app/views/bright_events/_filters.html.erb (hunk 1 above)
Comment (farellano0) [reply app/views/bright_events/_filters.html.erb]: Sorry, we will correct it as soon as possible.

File: app/jobs/event_reminder_job.rb (hunk 2 above)
Comment (farellano0) [reply app/jobs/event_reminder_job.rb]: Right away!

File: app/views/bright_events/_filters.html.erb (hunk 1 above)
Comment (farellano0) [reply app/views/bright_events/_filters.html.erb]: Done!

**Review by juperameza — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]

**Pull request merged**